packages/lisk-api-client/src/errors.ts

```diff
@@ -0,0 +1,11 @@
+import { VError } from 'verror';
+
+const defaultErrorNo = 500;
+
+export class APIError extends VError {
+	public errno?: number;
+	public constructor(message?: string, errno?: number) {
+		super(message || '');
+		this.errno = errno || defaultErrorNo;
+	}
```

Comment:
We should also override `name` property to something like `this.name = 'API Error'`. So, in this case, it will give out stack something like, `{ API Error: An unknown error occurred...` instead of `{ VError: An unknown error occurred...`